### ISSUE TITLE
Import configuration from Basic Logistics pipe to ItemSink module

### ISSUE THREAD

**Issue by Michal-MK** [RS485/LogisticsPipes#1043]
It would be nice if we could right/shift+right click a configured Basic Logistics pipe to copy the settings to the ItemSink module instead of having to find all the items again.
Thanks M.

**Comment by hron84** [comment]
Or maybe we need a tool similar to BC gate copier. It would be awesome if we could copy settings (filters, options, etc) between pipes/modules.

**Comment by SonOfTheStars** [comment]
For modules, you can configure up to 64 Modules at once(By simply RMB-Clicking with the Stack into the Air)
For Pipes id agree it would be lovely to have an option to Copy-Pasta settings between Pipes.

But i dont think there will be alot of changes made to LP in the near Future

**Comment by hron84** [comment]
I can imagine a scenario where you build a strucutre with chassis mk2-3 pipes, and then expand the system with same pipes. Copying from already-configured modules is not possible due it will not stack with the unconfigured module. The easiest (but most annoying) way to solve this is a custom crafting handler where you can put a configured and an unconfigured module to eachother, and the result will be a copied configuration. But this needs a very precise NBT checking.

**Comment by SonOfTheStars** [comment]
Well you can purge the config off of a module by crafting it with a stack of modules...and then reconfigure the whole stack at once

**Comment by Michal-MK** [comment]
My original idea was to copy settings from pipe to module.  Lets say you have a Trash can and 4 Basic LPs
connected to it. Currenty there is no easy way to make a chassi with 4 ItemSink modules from that. (other than find the 30+ items that are configured in the pipe)

**Comment by hron84** [comment]
@Bitterholz yeah, but no way to simply copy a working setup to an another module.

**Comment by hron84** [comment]
@MoonKill Trash can should have Terminus module instead. They are designed exactly for this purpose.

**Comment by Michal-MK** [comment]
@hron84 Well that depends on what you are voiding, but it was just to give an example on pipe to module copying  ;)

**Comment by SonOfTheStars** [comment]
Well ultimately, Logistics Pipes aint like AE, its not for the Lazy PPL.
I dont really see a specific need other than lazyness for this feature(no offense)
You cant config that much to a module, and since you can do 64 of them at once, this is really not needed

**Comment by hron84** [comment]
@Bitterholz all IT workers are lazy. I am working in the IT sector. :wink:

**Comment by SonOfTheStars** [comment]
@hron84 so am I my friend...so am I. But still im not lazy 😛

**Comment by Michal-MK** [comment]
The stacking feature is not helpful in this case if you need to void ~50 different items.  And I am NOT a fan if AE2 ;)  But I guess we can close this issue now.

**Comment by AEnterprise** [comment]
i agree LP isn't AE and neighter should it be, but there is a big difference between lazyness and doing the same same work over and over again.

also a hint for if you need setup a module again: you can middle click something in NEI while having the gui open to get a ghost stack you can use to configure the module/pipe

**Comment by dominic-jones** [comment]
Apologies for the necro but it's still open and tagged as needs comments, so..

I also +1 this type of feature. I would like to export Crafting Pipes to Crafting Modules when expanding the network. I would like to copy Active Supplier modules for creating new copies of "working materials" chests around my base.

**Comment by bziemons** [comment]
I also think it is useful. This issue should be migrated to the feature request wiki page.
